README.md

```diff
@@ -20,11 +20,26 @@ This terraform module creates an EC2 Instance with a security group, Cloudwatch
 - Follows aws security best practices and uses checkov to ensure compliance.
 - Has elaborate examples that you can use to setup your ec2 instance within a very short time.
 - Ability to create associated ec2 resources with minimum configuration changes.
+- This module includes a feature to install ssm agent and gives the necessary permissions for the instance to communicate with SSM manager.
+- As of [2.0.0] we no longer support or enable SSH keys on the instances, this is aligned with AWS best practices. As an alternative you should use Session Mananger which providers support to login to the Ec2 Linux instance and port-forwarding.
+
 
 Examples available [here](./examples/)
 
+## Connecting to the Instance
+- This module does not include a feature to create and use a key pair. As of this release this is only supported for linux instances
+- Use SSM Manager to connect to the instance. The module includes a feature to add the necessary permissions for the instance to communicate with systems manager.
+-
+## Launching in Private Subnets without NAT Gateways or internet connection
+To connect and manage instances in isolated subnets without internet connectivity, you need to enable VPC endpoints for specific services such as
+- `com.amazonaws.[region].ssm`
+- `com.amazonaws.[region].ec2messages`
+- `com.amazonaws.[region].ssmmessages`
+- `(optional) com.amazonaws.[region].kms for KMS encryption in Session Manager`
```

Comment:
Mention here the vpc-endpoints module ;)

Reply:
Done

Reply:
Done